sfepy/homogenization/engine.py

```diff
@@ -287,7 +309,27 @@ def __init__(self, num_workers):
     def __call__(self, problem, options, post_process_hook,
                  req_info, coef_info,
                  micro_coors, store_micro_idxs, chunk_size, time_tag=''):
-        """ !!! """
+        """Calculate homogenized correctors and coefficients.
+
+        Parameters
+        ----------
+        The same parameters as HomogenizationWorker, extended by:
```

Comment:
The same parameters as :class:`HomogenizationWorker`, extended by:

- please verify, that the link is correct in the html docs.